src/nodedef.cpp

```diff
@@ -1375,12 +1381,19 @@ void NodeDefManager::updateTextures(IGameDef *gamedef,
 
 	u32 size = m_content_features.size();
 
+	bool success = true;
+
 	for (u32 i = 0; i < size; i++) {
 		ContentFeatures *f = &(m_content_features[i]);
-		f->updateTextures(tsrc, shdsrc, meshmanip, client, tsettings);
+		if (!f->updateTextures(tsrc, shdsrc, meshmanip, client, tsettings))
+			success = false;
```

Comment:
Does this loop need to complete in case of failure?

Reply:
It doesn't - my bad. Thanks for pointing it out :) Fixed